src/world/area_trd/trd_03/9A4650.c

```diff
@@ -1,5 +1,22 @@
 #include "trd_03.h"
 
-INCLUDE_ASM(s32, "world/area_trd/trd_03/9A4650", func_80240000_9A4650);
+static ApiStatus func_80240000_9A4650(ScriptInstance* script, s32 isInitialCall) {
+    Bytecode* args = script->ptrReadPos;
+    u32 stickX;
 
-INCLUDE_ASM(s32, "world/area_trd/trd_03/9A4650", func_8024006C_9A46BC);
+    stickX = 0;
+    if (gCollisionStatus.pushingAgainstWall == 35) {
```

Comment:
Map shape and hit data have string names in them for all models and colliders.

Probably worth adding a comment that this collider is called `o84`, or even a `#define` in the header: once we have `trd_03_hit` data dumped we can generate these defines but it'd be good to avoid magic numbers even before that point.

Reply:
e.g.
```c
// trd_03.h

#define COLLIDER_o84 35
#define COLLIDER_o85 37
```

Reply:
(Note not all colliders/models are named `oXX`; some have descriptive romanji names; see `kmr_12/events.c` for reference. I need to set up similar `#define`s for kmr_12 too...)